src/mol-math/geometry/primitives/box3d.ts

```diff
@@ -130,6 +130,14 @@ namespace Box3D {
             v[2] < box.min[2] || v[2] > box.max[2]
         ) ? false : true;
     }
+
+    export function overlaps(a: Box3D, b: Box3D) {
+        return (
+            a.max[0] < b.min[0] || a.min[0] > b.max[0] ||
+            a.max[1] < b.min[1] || a.min[1] > b.max[1] ||
+            a.max[2] < b.min[2] || a.min[2] > b.max[2]
+        ) ? false : true;
```

Comment:
I am gonna assume this one is correct :)

```suggestion
        return !(
            a.max[0] < b.min[0] || a.min[0] > b.max[0] ||
            a.max[1] < b.min[1] || a.min[1] > b.max[1] ||
            a.max[2] < b.min[2] || a.min[2] > b.max[2]
        );
```

Reply:
:)